controllers/argocd_metrics_controller.go

```diff
@@ -529,7 +529,8 @@ func newPrometheusRule(namespace string) *monitoringv1.PrometheusRule {
 						},
 						For: "5m",
 						Labels: map[string]string{
-							"severity": "warning",
+							"severity":    "warning",
+							"description": "ArgoCDSyncAlert is listed if Argo CD app is out of sync",
```

Comment:
it could be interesting to include alert labels in the rendered string to help troubleshooting. For instance "Argo CD application XXX is out of sync. Check blah..."

Reply:
@simonpasquier addressed the changes, PTAL